Sources/Tracking/CustomerIO.swift

```diff
@@ -233,21 +192,14 @@ public class CustomerIO: CustomerIOInstance {
      This value can be an internal ID that your system uses or an email address.
      [Learn more](https://customer.io/docs/api/#operation/identify)
      - body: Request body of identifying profile. Use to define user attributes.
-     - onComplete: Asynchronous callback with `Result` of identifying a customer.
-     Check result to see if error or success. Callback called on main thread.
      - jsonEncoder: Provide custom JSONEncoder to have more control over the JSON request body for attributes.
      */
     public func identify<RequestBody: Encodable>(
         identifier: String,
         body: RequestBody,
-        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
         jsonEncoder: JSONEncoder? = nil
     ) {
-        guard let implementation = self.implementation else {
-            return onComplete(Result.failure(.notInitialized))
-        }
-
-        implementation.identify(identifier: identifier, body: body, onComplete: onComplete, jsonEncoder: jsonEncoder)
+        implementation?.identify(identifier: identifier, body: body, jsonEncoder: jsonEncoder)
```

Comment:
Just curious to learn that why `onComplete` completionHandler has been removed ?

Reply:
`onComplete` was used previously as a customer using the SDK must perform error handling if the HTTP request failed. Now that we have a background queue, the SDK does the error handling! Therefore, no need to asynchronously callback to the customer. They can simply give us the data and we handle it.